src/model/game/GameManager.ts

```diff
@@ -594,6 +595,12 @@ export default class GameManager {
             "https://thunderstore.io/c/content-warning/api/v1/package/", EXCLUSIONS,
             [new StorePlatformMetadata(StorePlatform.STEAM, "2881650")], "ContentWarning.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, []),
+
+        new Game("Balatro", "Balatro", "Balatro",
+            "Balatro", ["Balatro.exe"], "Balatro_Data",
+            "https://thunderstore.io/c/balatro/api/v1/package/", EXCLUSIONS,
+            [new StorePlatformMetadata(StorePlatform.STEAM, "2379780")], "Balatro.png",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.LOVELY, []),
```

Comment:
#1287 adds missing shorthands (the empty array at the end) for other entries here. IDK what they're used for or how one derives a shorthand for a game, but maybe Balatro should have one too?

Reply:
Mythic has said that these shorthands should be ignored and left empty 🤷 Didn't get the reason though

Reply:
They're just shorthands for searching on the game selection screen